index.js

```diff
@@ -20,6 +20,14 @@ function copy(text, options) {
     mark.textContent = text;
     // used to conserve newline, etc
     mark.style.whiteSpace = 'pre';
+
+    // position fixed because MS Edge scroll to upcoming seleceting node
+    // apart from this part it scrolls to the end of page
```

Comment:
Not sure what was meant here. Maybe, `following code prevents scrolling to the end of the page`?